templates/documents/copy.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
+{% block title %}{% trans %}Copy document to other cases{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Basic information{% endtrans %}</h1>
+        </div>
+        <div class="col-auto">
+            <a name="" id="" class="btn btn-secondary" href="{{ url('case_documents', {id: case.id}) }}" role="button">{% trans %}Cancel{% endtrans %}</a>
+        </div>
+    </div>
+
+    {{ form_start(copy_document_form) }}
+    <form class="form" >
```

Comment:
Will this not produce two `<form>` tags?

Reply:
Yes. Avoided duplicate `<form>` elements.